src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.ts

```diff
@@ -1,25 +1,34 @@
 import { Component, OnInit, Input, OnDestroy } from '@angular/core';
 import { ITimelineData, TimeLineGeneratorBase, parseEventsGenerically } from 'src/app/Models/eventstore/timelineGenerators';
-import { EventListBase } from 'src/app/Models/DataModels/collections/Collections';
-import { FabricEventBase } from 'src/app/Models/eventstore/Events';
 import { TimeUtils } from 'src/app/Utils/TimeUtils';
 import { IOnDateChange } from '../double-slider/double-slider.component';
-import { Subject, Subscription, forkJoin } from 'rxjs';
-import { debounceTime, distinctUntilChanged, finalize } from 'rxjs/operators';
+import { Subject, Subscription, forkJoin, of } from 'rxjs';
+import { debounceTime, distinctUntilChanged, finalize, subscribeOn } from 'rxjs/operators';
 import { DataService } from 'src/app/services/data.service';
 import { hostViewClassName } from '@angular/compiler';
 import { DataGroup, DataItem, DataSet } from 'vis-timeline/standalone/esm';
+import { Observable } from 'rxjs';
+import { DataModelCollectionBase } from 'src/app/Models/DataModels/collections/CollectionBase';
+import { ListSettings } from 'src/app/Models/ListSettings';
 
 export interface IQuickDates {
     display: string;
     hours: number;
 }
 
-export interface IEventStoreData {
-    eventsList: EventListBase<any>;
-    timelineGenerator?: TimeLineGeneratorBase<FabricEventBase>;
+export interface IEventStoreDataUtils{
```

Comment:
this is being removed in a later PR correct?

Reply:
Yep, we are combining everything in just one interface.